ModelViewIntentSample/core/src/main/java/com/novoda/movies/mvi/search/BaseStore.kt

```diff
@@ -6,10 +6,10 @@ import io.reactivex.subjects.BehaviorSubject
 import io.reactivex.subjects.PublishSubject
 
 class BaseStore<A, S, C>(
-    private val schedulingStrategy: SchedulingStrategy,
-    private val reducer: Reducer<S, C>,
-    private val middlewares: List<Middleware<A, S, C>>,
-    private val initialValue: S
+        private val schedulingStrategy: SchedulingStrategy,
```

Comment:
What do I need to have a consistent indenting with the rest of the project?

Reply:
Did you import the novoda code styles from https://github.com/novoda/novoda/tree/master/ide-settings/intellij ?